src/components/Posts/ExpandedPostItem.tsx

```diff
@@ -124,13 +123,7 @@ const ExpandedPostItem = ({ post }: ExpandedPostItemProps) => {
       </Box>
       <CardHeader
         avatar={<UserAvatar username={post.username} />}
-        action={
-          <PostMenu
-            authorId={post.userId}
-            postId={post.postId}
-            isExpandedPost
-          />
-        }
+        action={<PostMenu authorId={post.userId} isExpandedPost post={post} />}
```

Comment:
To avoid redundancy and since we are already passing the entire post object, let's remove the `authorId` and use `post.userId` within the component instead

Reply:
Great idea, I'll do this!